npm/init.js

```diff
@@ -16,15 +17,30 @@ function validateNpmVersion() {
     }
 }
 
+function download(url) {
+    // We detect proxy by looking at the environment variable
+    var agent = https;
+    if(process.env.https_proxy && process.env.https_proxy.length > 0) {
+        agent = http;
+        url = process.env.https_proxy + starturl.replace('https://', '/https/');
+    }
+
+    agent.get(url, function(res) {
+        if(res.statusCode == 301 || res.statusCode == 302) {
```

Comment:
Can you please replace
```if(res.statusCode == 301...``` with ```if (res.statusCode == 301...```
(notice the missing white-space) to remain compatible with the code styling in this file?

Reply:
done